bin/workflows/pycbc_make_offline_search_workflow

```diff
@@ -187,8 +188,76 @@ insps = wf.merge_single_detector_hdf_files(workflow, hdfbank,
                                            insps, output_dir,
                                            tags=['full_data'])
 
+# Check for the only-do section. If inspiral=true then it will only do the workflow up until the merged inspiral hdf files
+if 'only-do' in workflow.cp.sections():
```

Comment:
The option may need a better name so it is clear in the config file. Maybe something more verbose?